corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -4,8 +4,9 @@ if (typeof cloudCare === 'undefined') {
 
 cloudCare.EMPTY = '---';
 
-var _caseListLoadError = function (response) {
+var _caseListLoadError = function (model, response) {
```

Comment:
sorry if this should be obvious, but why is the signature changing? was it just wrong?